File: 8kyu 2/getPlanetName.js
function getPlanetName(id) {
   let name;
   switch (id) {
      case 1:
         name = 'Mercury'
         break
      case 2:
         name = 'Venus'
         break
      case 3:
         name = 'Earth'
         break
      case 4:
         name = 'Mars'
         break
      case 5:
         name = 'Jupiter'
         break
      case 6:
         name = 'Saturn'
         break
      case 7:
         name = 'Uranus'
         break
      case 8:
         name = 'Neptune'
         break
   }
   return name;
}


function getPlanetName(i) {
   return (['Mercury', 'Venus', 'Earth', 'Mars', 'Jupiter', 'Saturn', 'Uranus', 'Neptune'])[i - 1];
}


function getPlanetName(id) {
   return {
      1: 'Mercury',
      2: 'Venus',
      3: 'Earth',
      4: 'Mars',
      5: 'Jupiter',
      6: 'Saturn',
      7: 'Uranus',
      8: 'Neptune'
   } [id]
}


console.log(getPlanetName(2), 'Venus');
console.log(getPlanetName(5), 'Jupiter');
console.log(getPlanetName(3), 'Earth');

File: Python/8py 1/get_planet_name.py
"""
|--------------------------------------------------------------------------
| Get Planet Name By ID
|--------------------------------------------------------------------------
|
"""

def get_planet_name(id):
   return {
      1 : 'Mercury',
      2 : 'Venus',
      3 : 'Earth',
      4 : 'Mars',
      5 : 'Jupiter',
      6 : 'Saturn',
      7 : 'Uranus',
      8 : 'Neptune',
   }.get(id, '')


def get_planet_name(id):
   return ["Mercury","Venus","Earth","Mars","Jupiter","Saturn","Uranus","Neptune"][id-1]


def get_planet_name(id):
   planets = {
      1: "Mercury",
      2: "Venus",
      3: "Earth",
      4: "Mars",
      5: "Jupiter",
      6: "Saturn",
      7: "Uranus", 
      8: "Neptune",
   }
   return planets[id]


def get_planet_name(id):
   return [0, "Mercury", "Venus", "Earth", "Mars", "Jupiter", "Saturn", "Uranus", "Neptune"][id]


print(get_planet_name(2), 'Venus')
print(get_planet_name(5), 'Jupiter')
print(get_planet_name(3), 'Earth')
print(get_planet_name(4), 'Mars')
print(get_planet_name(8), 'Neptune')
print(get_planet_name(1), 'Mercury')